ideascube/serveradmin/wifi.py

```diff
@@ -0,0 +1,183 @@
+from itertools import groupby
+from operator import attrgetter
+from uuid import uuid4
+
+from django.utils.translation import ugettext as _
+
+try:
+    from NetworkManager import (
+        NetworkManager, Settings as NMSettings, NM_DEVICE_TYPE_WIFI)
+
+except Exception as e:
+    import sys
+    sys.stderr.write("Could not import the NetworkManager module: %s\n" % e)
+    sys.stderr.write("Wifi management will not work properly\n")
+    sys.stderr.flush()
+
```

Comment:
I think we need this to be a bit more friendly, with a flag like `wifi_available`, that we'd check to display or not the wifi link in the admin and to call or not wifi related code.